lightning/src/ln/channelmanager.rs

```diff
@@ -1105,8 +1105,8 @@ impl ChannelDetails {
 }
 
 /// Used by [`ChannelManager::list_pending_payments`] to express the status of payments that are
-/// currently pending resolution. These not only include payments that are currently inflight, but
-/// also those that may have yet to find a successful path or have outstanding HTLCs.
+/// currently pending. These not only include payments that are currently inflight, but also those
```

Comment:
Not sure if we want to use "pending" here given that is one of the variants. Probably can just say "status of recent payment" given the method rename.